examples/grpc-service-reactor/src/test/java/example/armeria/grpc/reactor/HelloServiceTest.java

```diff
@@ -0,0 +1,225 @@
+
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.LinkedBlockingQueue;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import com.google.common.base.Stopwatch;
+import com.google.common.util.concurrent.FutureCallback;
+import com.google.common.util.concurrent.Futures;
+import com.google.common.util.concurrent.ListenableFuture;
+import com.google.common.util.concurrent.MoreExecutors;
+
+import com.linecorp.armeria.client.Clients;
+import com.linecorp.armeria.server.Server;
+
+import example.armeria.grpc.reactor.Hello.HelloReply;
+import example.armeria.grpc.reactor.Hello.HelloRequest;
+import io.grpc.stub.StreamObserver;
+
+public class HelloServiceTest {
+
+    private static Server server;
+    private static HelloServiceGrpc.HelloServiceStub helloService;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        server = Main.newServer(0, 0);
+        server.start().join();
+        helloService = Clients.newClient(uri(), HelloServiceGrpc.HelloServiceStub.class);
+    }
+
+    @AfterClass
+    public static void afterClass() {
+        if (server != null) {
+            server.stop().join();
+        }
+    }
+
+    private static String uri() {
+        return "gproto+http://127.0.0.1:" + server.activeLocalPort() + '/';
+    }
+
+    @Test
+    public void getReply() {
+        final HelloServiceGrpc.HelloServiceBlockingStub helloService = Clients.newClient(uri(), HelloServiceGrpc.HelloServiceBlockingStub.class);
+        assertThat(helloService.hello(HelloRequest.newBuilder().setName("Armeria").build()).getMessage())
+                .isEqualTo("Hello, Armeria!");
+    }
+
+    @Test
+    public void getReplyWithDelay() {
+        final HelloServiceGrpc.HelloServiceFutureStub helloService = Clients.newClient(uri(), HelloServiceGrpc.HelloServiceFutureStub.class);
+        final ListenableFuture<HelloReply> future =
+                helloService.lazyHello(HelloRequest.newBuilder().setName("Armeria").build());
+        final AtomicBoolean completed = new AtomicBoolean();
+        Futures.addCallback(future, new FutureCallback<HelloReply>() {
+            @Override
+            public void onSuccess(HelloReply result) {
+                assertThat(result.getMessage()).isEqualTo("Hello, Armeria!");
+                completed.set(true);
+            }
+
+            @Override
+            public void onFailure(Throwable t) {
+                // Should never reach here.
+                throw new Error(t);
+            }
+        }, MoreExecutors.directExecutor());
+
+        await().untilTrue(completed);
+    }
+
+    @Test
+    public void getReplyFromServerSideBlockingCall() {
+        final HelloServiceGrpc.HelloServiceBlockingStub helloService = Clients.newClient(uri(), HelloServiceGrpc.HelloServiceBlockingStub.class);
```

Comment:
nit: Consider breaking the line after ` = `.

Reply:
done